.github/workflows/pkgcheck.yml

```diff
@@ -8,14 +8,37 @@ jobs:
       fail-fast: false
       matrix:
         name: [
-          Ubuntu GCC
+          Ubuntu GCC,
+          Ubuntu GCC AARCH64,
+          Ubuntu GCC PPC
         ]
         include:
           - name: Ubuntu GCC
             os: ubuntu-latest
             compiler: gcc
             packages: ninja-build
 
+          - name: Ubuntu GCC AARCH64
+            os: ubuntu-latest
+            chost: aarch64-linux-gnu
+            compiler: aarch64-linux-gnu-gcc
+            cmake-args: -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain-aarch64.cmake
+            packages: ninja-build qemu gcc-aarch64-linux-gnu libc-dev-arm64-cross
+
+          - name: Ubuntu GCC PPC
+            os: ubuntu-latest
+            chost: powerpc-linux-gnu
+            compiler: powerpc-linux-gnu-gcc
+            cmake-args: -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain-powerpc.cmake
+            packages: ninja-build qemu gcc-powerpc-linux-gnu libc-dev-powerpc-cross
```

Comment:
glibc packages on Ubuntu seem to start with `libc6`. So this should be `libc6-dev-powerpc-cross`

Reply:
Yes, looks like the libc- one is an alias, but it's always less confusing to use the real package name.